src/api/contributionsApi.test.ts

```diff
@@ -26,15 +27,17 @@ const epicResponse = {
 };
 
 beforeEach(() => {
-    clearCachedEpic();
```

Comment:
Now we've removed this, we could if we wanted replace this whole block with:

```ts
beforeEach(fetchMock.resetHistory);
```

(`fetchMock` does the right thing re binding to ensure this works correctly).